src/lib/features/change-request-access-service/sql-change-request-access-read-model.ts

```diff
@@ -35,6 +35,23 @@ export class ChangeRequestAccessReadModel
         return !(changeRequestEnabled && !canSkipChangeRequest);
     }
 
+    public async canBypassChangeRequestForProject(
+        project: string,
+        user?: User,
+    ): Promise<boolean> {
+        const [canSkipChangeRequest, changeRequestEnabled] = await Promise.all([
+            user
+                ? this.accessService.hasPermission(
+                      user,
+                      SKIP_CHANGE_REQUEST,
+                      project,
+                  )
+                : Promise.resolve(false),
+            this.isChangeRequestsEnabledForProject(project),
+        ]);
+        return !(changeRequestEnabled && !canSkipChangeRequest);
```

Comment:
Could we express this without the double negation? I'm struggling to parse what we're saying here.

Reply:
great catch, changing it to  `return canSkipChangeRequest || !changeRequestEnabled;`